Examples/Examples-iOS/IGListKitExamples/Views/DetailLabelCell.swift

```diff
@@ -38,6 +38,19 @@ final class DetailLabelCell: UICollectionViewCell {
         return view
     }()
 
+
+    var title: String? {
+        didSet {
+            titleLabel.text = title
+        }
+    }
```

Comment:
i'm not a fan of these changes on all of the cells. the properties are redundant, and create unnecessary storage.

if we do something like this, they should be computed, not stored:

```swift
var title: String? {
    get { 
        return titleLabel.text
    }
    set {
        titleLabel.text = newValue
    }
}
```